build.sh

```diff
@@ -225,6 +226,12 @@ do_build() {
     image_path="output/vzlinux"
     [[ ! -d image_path ]] || rm -rf image_path
     ;;
+  openvz-7 )
```

Comment:
Is there an extra space after `openvz-7`?

Reply:
Yep, I just copy `vzlinux-8` section. Will remove it.